docs/UserGuide.md

```diff
@@ -377,27 +382,32 @@ Sorts the expenditure list by ascending or descending amount, or from earliest t
 
 Sorts the expenditure list by ascending or descending amount, or from earliest to latest date added. It will then display the sorted expenditure list.
 
-**Format:** `viewdate DATE`
+**Format:** `viewdate DATE CURRENCY`
```

Comment:
Good keep this up! we should aim to update the DG and UG as we update the command parameters so that we will not miss anything out